Command/FixturesLoadCommand.php

```diff
@@ -204,7 +204,7 @@ protected function loadFixtures(InputInterface $input, OutputInterface $output,
      */
     protected function loadSqlFixtures(InputInterface $input, OutputInterface $output)
     {
-        $tmpdir = $this->getApplication()->getKernel()->getRootDir() . '/cache/propel';
+        $tmpdir = $this->getApplication()->getKernel()->getProjectDir() . '/var/cache/propel';
```

Comment:
Is this change in location intentional?
`./app/cache/propel` --> `./var/cache/propel`

Reply:
It is, seemed better to me to group generated files together in a subdir than in the root dir.